src/cgame/cg_event.cpp

```diff
@@ -473,52 +472,44 @@ static void CG_Obituary( entityState_t *ent )
 				break;
 
 			case MOD_LEVEL1_CLAW:
-				message = G_( "%s%s ^*was sliced by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was sliced by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL1;
+				message = G_( "%s%s ^*was sliced by %s%s%s^*'s claws" );
+				messageAssisted = G_( "%s%s ^*was sliced by %s%s%s^*'s claws; %s%s%s^* assisted" );
 				break;
 
 			case MOD_LEVEL2_CLAW:
-				message = G_( "%s%s ^*was shredded by %s%s%s^*'s %s" );
-				messageAssisted = G_( "%s%s ^*was shredded by %s%s%s^*'s %s^*; %s%s%s^* assisted" );
-				attackerClass = PCL_ALIEN_LEVEL2;
+				message = G_( "%s%s ^*was shredded by %s%s%s^*'s claws" );
+				messageAssisted = G_( "%s%s ^*was shredded by %s%s%s^*'s claws; %s%s%s^* assisted" );
 				break;
 
 			case MOD_LEVEL2_ZAP:
-				message = G_( "%s%s ^*was electrocuted by %s%s%s^*'s %s" );
```

Comment:
if you remove this, do you have to remove an argument elsewhere?

Reply:
Yes, the (now removed) `attackerClass` class variable and the (now removed) `if/else` code to duplicate the log messaging with it.

Reply:
This happens below in the diff.